src/ui/common/src/components/workflows/artifact/summaryList.tsx

```diff
@@ -15,80 +15,53 @@ type Props = {
   workflowId: string;
   dagResultId: string;
   artifactResults: ArtifactResultResponse[];
-  initiallyExpanded: boolean;
-};
-
-const listStyle = {
-  width: '100%',
-  maxWidth: 360,
-  bgcolor: 'background.paper',
 };
 
 const SummaryList: React.FC<Props> = ({
   title,
   workflowId,
   dagResultId,
   artifactResults,
-  initiallyExpanded,
 }) => {
-  const items = artifactResults.map((artifactResult, index) => {
-    let content = null,
-      link = null;
-    if (artifactResult.result?.content_serialized) {
-      content = artifactResult.result.content_serialized;
-    } else {
-      link = `${getPathPrefix()}/workflow/${workflowId}/result/${dagResultId}/artifact/${
-        artifactResult.id
-      }}`;
-      content = artifactResult.name;
-    }
-
-    const element = (
-      <Box
+  const items = artifactResults.map((artifactResult, idx) => {
```

Comment:
@vsreekanti I just realized you already modified this component in #543 . I don't think I make much appearance changes. The main changes is to additionally make numeric artifacts a link which I think is important to enable us to traverse in the dag using inputs / outputs section.